src/plugins/kibana_react/public/code_editor/editor_theme.ts

```diff
@@ -101,6 +101,11 @@ export function createTheme(
       'editor.selectionBackground': selectionBackgroundColor,
       'editorWidget.border': euiTheme.euiColorLightShade,
       'editorWidget.background': euiTheme.euiColorLightestShade,
+      'editorCursor.foreground': euiTheme.euiColorDarkestShade,
+      'editorSuggestWidget.selectedBackground': euiTheme.euiColorLightShade,
+      'list.hoverBackground': euiTheme.euiColorLightShade,
```

Comment:
```suggestion
      'list.hoverBackground': euiTheme.euiFocusBackgroundColor,
```

Reply:
This changes makes highlighting hardly visible in dark mode:


![image](https://user-images.githubusercontent.com/31325372/75662192-bd9ce700-5c7f-11ea-9261-a74f334228f7.png)

Reply:
Ah good catch, yeah I think our focus background color isn't so good in dark mode. Just leave as light shade for now then.